asreview/state/sql_converter.py

```diff
@@ -62,7 +62,7 @@ def convert_asreview(fp):
 
     # Create the reviews folder and the paths for the results and settings.
     Path(fp, 'reviews', review_id).mkdir(parents=True)
-    sql_fp = Path(fp, 'reviews', review_id, 'results.sql')
+    sql_fp = str(Path(fp, 'reviews', review_id, 'results.sql'))
```

Comment:
Any particular reason for turning this into a string?

Reply:
`sqlite3.connect()` does not accept a `PosixPath` argument.

Reply:
Ah okay! This seems to be solved in python 3.7 (https://bugs.python.org/issue33496), but then it's probably smart to keep it a string.

Reply:
Hmm... then I guess our test workflow needs an update. @J535D165 
https://github.com/asreview/asreview/blob/master/.github/workflows/ci-frontend.yml#L10